packages/debug-test/PlaygroundAutomation.ts

```diff
@@ -0,0 +1,234 @@
+// Notes about WebDriverIO/WinAppDriver usage in this file:
+//
+// It seems that the WebDriverIO/WinAppDriver combination is buggy. Among the issues discovered are:
+// - WinAppDriver does not support the search strategies resulting from some selector expressions,
+//   resulting in errors like
+//       ERROR webdriver: Request failed with status 501 due to Error: Unimplemented Command: css selector locator strategy is not supported
+// - When using the findElementByXPath method, WebdriverIO (WDIO) finds UI elements after they have been removed.
+//   This happens despite WinAppDriver responding with seemingly correct 404 "element not found"
+//   errors. The returned UI elements can be queried for their text property, suggesting they represent 
+//   WDIO cache state.
+// - XPath expressions that yield expected results in $$() selector expressions result in "invalid elementId"
+//   errors when used in $() selector expressions.
+// - WDIO does not appear to recognize XPath expressions unless they start with either "//" or "./"
+// At the time of this writing, the most reliable/predictable way of finding elements seems to be $$() selector
+// expressions with XPath arguments. Those expressions return arrays which is the reason why many UI element
+// lookups in this file are conducted "in the plural" using array index index expressions.
```

Comment:
Is it possible to fold this into a friendly facade in `AutomationClient`? My historical intent with that interface was to shim away some of the less polished bits of the WebDriver/WinAppDriver interaction (e.g. APIs designed for web that fail immediately), to expose a more friendly API.

I had tried to keep WebDriverIO global APIs private-ish to the automation framework. This is mainly so that the API is discoverable without looking up the globals, and to reduce direct usage of third-party APIs, in case we want to replace WebdriverIO as an implementation detail.

I think you will also see `eslint` warnings about undefined globals in this project once that is set up,

Reply:
What do you mean with "AutomationClient"? packages\@react-native-windows\automation\lib-commonjs\AutomationClient.js?

If so, the code in PlaygroundAutomation.ts is, from a layering perspective, above that. It targets the GUI of a specific app (Playground UWP app). Similar to the intentions you express above, the code in PlaygroundAutomation.ts exports only "semantic" constructs (e.g. select a package in the Playground app) while encapsulating the details of how this is achieved (e.g. enter the name of the desired package in a combo box and click the "Load" button).

If this doesn't line up with what you meant, can you please elaborate on what you want me to do?

Reply:
AutomationClient.ts for the file. I think this change already has an edit there.

I should have been a bit more specific as to what I was pointing to. If there is a specific reliable way to find elements via XPath, it seems valuable to expose it in the library code, for other automation using code. The design, of going through AutomationClient instead of Webdriverio directly is enforced in packages which have linting enabled. "$$" works at runtime, but will be flagged by the linter as disallowed.

Reply:
Is your ask then to move 
    packages\debug-test\PlaygroundAutomation.ts
into
   packages\@react-native-windows\automation\src\
? If so, I'd prefer to do that as a separate change.